bin/project-rk.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/env bash
+
+export C=$WINEPREFIX/drive_c
+
+# Gens wineprefix on first start
+(
+  if [[ ! -d $WINEPREFIX ]]; then
+    umu-run winetricks win7 dxvk corefonts
+  fi
+) | zenity --progress --title="Project Rubi-Ka" --text="Installing and configuring prefix..." --pulsate --auto-close --percentage=0 --no-cancel --width=360 --height=90
+
+# Gens game launcher if missing/deleted
+(
+  if [[ ! -d $C/launcher ]]; then
+    mkdir -p $C/launcher
+    curl -sL https://prk-vault.nyc3.digitaloceanspaces.com/launcher/win.zip -o $C/win.zip
+    unzip $C/win.zip -d $C/launcher && rm $C/win.zip
```

Comment:
why does this need to be downloaded at runtime? Why can it not be in the manifest?

Reply:
also you are downloading without doing any king of verification from a link that is not versioned, this is not very secure or safe.

Reply:
It self-updates, and the data has to reside in a wineprefix, which is not immutable and can't interop with read only fs.

Reply:
the solution would be to not make it self update and do updates through flatpak.

Reply:
We literally can't do that from a tech perspective, wineprefixes generate unique ids, and the application resides in that. the mutable data can't be treated as immutable.

We will be continuing to maintain the part in the flatpak that matters here, which you seem to be overlooking- which is the freedesktop layers and umu/steam runtime layers that give us a standardized environment to test against and support.

Reply:
This is a very common practice-stuffing many many gigabytes of files into a flatpak is also not a good idea; most games do self-hydration like this...

Reply:
then the link should be versioned, and some hash verification needs to be done before extracting.

Reply:
also, it would be better if it came from the prject-rk website instead of a cdn URL. Because I have no way to verify who controls that.

Reply:
I'll add the sha256sum checking to protect the end user

Reply:
done!